.github/workflows/lint.yaml

```diff
@@ -79,6 +79,16 @@ jobs:
         run: |
           source venv/bin/activate
           check-jsonschema --check-metaschema schema/dataKeys.json
+  lint-jsonschema:
+    name: Lint json schema file
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v4
+      - name: Run json schema lint
+        run: |
+          set -e
+          diff -u schema/dataKeys.json <(jq . schema/dataKeys.json) 
```

Comment:
Maybe include some useful message if they differ? Like, what is the easiest way to format it locally? Since `jq` does not support edits in place, you would need a temp file. Is there an easier way?